pkg/models/v1/models.go

```diff
@@ -176,24 +176,15 @@ func AppendAdminEmailLog(vasp *pb.VASP, reason string, subject string) (err erro
 	return nil
 }
 
-// VASPEmailEntry represents a log entry relevant to a single contact. It embeds an
-// email log entry and adds contact-specific information so that higher level methods
-// reading a unified contact log can distinguish between entries for different contacts.
-// TODO: This metadata should really be stored in the email log entry itself.
-type VASPEmailEntry struct {
-	*EmailLogEntry
-	ContactType string
-}
```

Comment:
We were previously using this intermediate struct but now that we have the email address in the log this is no longer necessary.